### PR TITLE
Add missing notebooks & summaries to READMEs; deleting old htmls

### PR DESCRIPTION
**Pull request by olivialynn**
For #182.

Opted to keep and update the READMEs instead of deleting them; the short summaries that were initially there struck me as nice and worth keeping around.

Links now point to notebook renders in the RTDs.

### CODE REVIEW

File: examples/creation/README.md [hunk 1]
```diff
@@ -2,4 +2,14 @@ This directory contains example notebooks explaining how to use the RAIL Creatio
 
 - [degradation-demo.ipynb](https://htmlpreview.github.io/?https://github.com/LSSTDESC/RAIL/blob/master/examples/creation/degradation-demo.html) explains how to construct a basic `Engine`, draw samples, and use Degraders to degrade the samples in various ways.
 
+- **dsps_sed_demo.ipynb** demonstrates some basic usage of the DSPS library.
+
+-[example_GridSelection_for_HSC.ipynb](https://lsstdescrail.readthedocs.io/en/latest/source/creation-notebooks.html#grid-selection-degrader-to-emulate-hsc-training-samples) creates a grid of mock galaxies and plots the success rate to visualize the spectroscopic success rate for HSC.
```
Comment (drewoldag) [inline examples/creation/README.md]:
```suggestion
- [example_GridSelection_for_HSC.ipynb](https://lsstdescrail.readthedocs.io/en/latest/source/creation-notebooks.html#grid-selection-degrader-to-emulate-hsc-training-samples) creates a grid of mock galaxies and plots the success rate to visualize the spectroscopic success rate for HSC.
```

Comment (drewoldag) [reply examples/creation/README.md]: nit: I noticed that without the space this renders as:

- dsps_sed_demo.ipynb demonstrates some basic usage of the DSPS library.

-[example_GridSelection_for_HSC.ipynb](https://lsstdescrail.readthedocs.io/en/latest/source/creation-notebooks.html#grid-selection-degrader-to-emulate-hsc-training-samples) creates a grid of mock galaxies and plots the success rate to visualize the spectroscopic success rate for HSC.

- example_ObsConditions.ipynb generates the photometric error using the Rubin Observatory Metrics Analysis Framework (MAF).

File: examples/goldenspike/README.md [hunk 2]
```diff
@@ -1,3 +1,3 @@
-This directory contains an example notebook showing a simplified version the end-to-end functionality of RAIL.
+This directory contains an example notebook showing a simplified version of the end-to-end functionality of RAIL.
 
-- [goldenspike.ipynb](https://htmlpreview.github.io/?https://github.com/LSSTDESC/RAIL/blob/master/examples/goldenspike/goldenspike.html) This notebooks chains together the functionality of the creation, estimation, and evaluation modules.
+- [goldenspike.ipynb](https://htmlpreview.github.io/?https://github.com/LSSTDESC/RAIL/blob/master/examples/goldenspike/goldenspike.html) is a notebook that chains together the functionality of the creation, estimation, and evaluation modules.
```
Comment (drewoldag) [inline examples/goldenspike/README.md]: I know that you didn't change the URL here, but I'm unfamiliar with the this one, and I'm not seeing it resolve. I've tried a couple ways shown below. Of course feel free to ignore if this is out of the scope of the ticket.

- https://htmlpreview.github.io/?https://github.com/LSSTDESC/RAIL/blob/master/examples/goldenspike/goldenspike.html
- https://github.com/LSSTDESC/RAIL/blob/main/examples/goldenspike/goldenspike.html

File: examples/estimation/README.md [hunk 3]
```diff
@@ -1,4 +1,9 @@
 This directory contains example notebooks explaining how to use the RAIL Estimation Module.
 
+- [nzdir_as_pipeline](https://lsstdescrail.readthedocs.io/en/latest/source/estimation-notebooks.html#the-nzdir-estimator) provides a quick demo of the NZDir estimator.
+
 - [RAIL_estimation_demo.ipynb](https://htmlpreview.github.io/?https://github.com/LSSTDESC/RAIL/blob/master/examples/estimation/RAIL_estimation_demo.html) explains to inform a model used for photo-z estimation, and then how to use that model to estimate p(z).
```
Comment (drewoldag) [inline examples/estimation/README.md]: I get a 404 when trying to follow this link. Is this considered in scope for this ticket?

**Review by drewoldag — APPROVED** [review]
First of all, my apologies for taking so long to review this. I'm sorry about that. 

All in all this looks good. I had a couple comments about links, and noticed one bullet point missing a space. Thanks for cleaning this up!

File: examples/creation/README.md (hunk 1 above)
Comment (olivialynn) [reply examples/creation/README.md]: Nice catch, thanks!

File: examples/goldenspike/README.md (hunk 2 above)
Comment (olivialynn) [reply examples/goldenspike/README.md]: Thanks--forgot to update this one with a link to the RTD notebooks!
